### PR TITLE
Turn into UPM package

### PR DESCRIPTION
**Pull request by johanhelsing**
This makes it much simpler to install in a project.

To test, you can add this git url through the unity package manager:

https://github.com/johanhelsing/sentry-unity.git#upm-package

The demo/sample can now be imported through the package manager interface

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "io.sentry.sentry-unity",
+  "version": "0.0.5",
+  "displayName": "Sentry Unity",
+  "description": "Sentry SDK for Unity",
+  "unity": "2020.1",
```
Comment (johanhelsing) [inline package.json:6]: FYI, this is probably higher than in needs to be, but I've only tested with 2020.1 so far.

Comment (bruno-garcia) [reply package.json:6]: I'll try it out on an older version then

**Review by bruno-garcia — APPROVED** [review]
Thanks a lot for this!

I'll test on 2018 first and try to lower the requirement

I'm hoping to phase out this version in favour of a new one though.
But would like to take this approach for installation too on the other one. It was on the todo list. 

A sneak peak video I recorded today: https://www.youtube.com/watch?v=bHc4QMlV8bM

I'd really appreciate feedback on this. If you'd like to get involved in the development and help guide the direction, feel free to reach out. I'm on [Sentry's Discord #unity channel](https://discord.gg/sentry) or [twitter](https://twitter.com/brungarc)

File: package.json (hunk 1 above)
Comment (johanhelsing) [reply package.json]: Lowered it to 2018.1. Tested and confirmed working on 2018.2, I don't have a 2018.1 install, though, but think it should work there as well.

**Comment by bruno-garcia** [comment]
Thanks @johanhelsing

I'll move this code to another repository though first.

**Pull request merged**